torch/csrc/jit/passes/python_print.cpp

```diff
@@ -720,12 +719,14 @@ struct PythonPrintPass {
         value->writeScalars(stmt);
         printValueList(stmt, node->inputs(), "(", ")");
       } break;
-      case prim::Constant: {
-        IValue v = toIValue(node->output()).value();
-        printConstant(stmt, v);
-      } break;
-      case prim::Undefined:
-      case prim::None: {
+      case prim::Constant:
```

Comment:
The conditional break is not easy to see. Can we add a emitNone function so it is clear that we have established the constant is none.